README.md

```diff
@@ -2,7 +2,8 @@
 
 A [collection of common configurations](doc/server_configurations.md) for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
 
-This plugin allows for declaratively configuring, launching, and initializing language servers you have installed on your system. Language server configurations are community-maintained.
+This plugin allows for declaratively configuring, launching, and initializing language servers you have installed on your system. 
+** Disclaimer: Language server configurations are on a best-effort basis and community-maintained. See (#contributions).**
```

Comment:
On a best basis? Feels like missing word

Reply:
oops, yes

Reply:
"best-effort basis" -- what exactly is missing?

Reply:
A past tense verb :)

Reply:
OK, I thought about "provided" as well ;) 

(I opted for conciseness instead.)